pages/linux/winetricks.md

```diff
@@ -0,0 +1,16 @@
+# winetricks
+
+> Manage Wine virtual Windows environments.
+> More information: <https://wiki.winehq.org/Winetricks>.
+
+- Start a graphical setup at the default wine location:
+
+`winetricks`
+
+- Specify a wine prefix to run winetricks in:
+
+`WINEPREFIX={{/path/to/wine/prefix}} winetricks`
```

Comment:
```suggestion
`WINEPREFIX={{path/to/wine/directory}} winetricks`
```

Reply:
Can the WINEPREFIX actually be relative? Just asking because I don't know.

Reply:
I have no idea, it's just what we always do.  We should discuss adding this to the contributing guidelines sometime...:thinking:

Reply:
Based on the Wiki I have it can be relative:
```bas
env WINEPREFIX=~/.winetest sh winetricks mfc40 
```

about a page with a command that requires a absolute path, we've an example here: https://github.com/tldr-pages/tldr/blob/master/pages/linux/ksvgtopng5.md